contrib/compare_ids_test.bzl

```diff
@@ -76,5 +80,10 @@ compare_ids_test = rule(
             single_file = True,
             default = "compare_ids_test.sh.tpl",
         ),
+        "_id_extract_script": attr.label(
+            allow_files = True,
+            single_file = True,
+            default = "extract_image_id.sh",
```

Comment:
Can you try to reference this file from base-image-docker repo. you might need to give the whole repo and package path here: like @ io_bazel_rules_docker//contrib:extract_image_id.sh

Reply:
Discussed offline. No need to change.